projects/addon-charts/components/bar-set/bar-set.template.html

```diff
@@ -1,7 +1,8 @@
 <ng-container *ngIf="size; else flexible">
     <ng-container *ngFor="let item of computedValue; let index = index">
         <ng-container
-            *ngTemplateOutlet="bar; context: {$implicit: item, index: index, flexible: false}"
+            [ngTemplateOutlet]="bar"
```

Comment:
In such cases this could go on the `ng-container` above, with `*ngFor` so there's less nesting. Please look through all such places where I've commented before